BLOG.adoc

```diff
@@ -307,7 +324,7 @@ The Loki Appender configuration looks exactly the same.
         </http>
         <format>
             <label>
-                <pattern>app=${appName},host=${HOSTNAME},traceID=%X{traceId:-NONE},level=%level</pattern>
+                <pattern>app=${appName},host=${HOSTNAME},level=%level,traceId=%X{traceId:-00000000000000000000000000000000}</pattern>
```

Comment:
As far as I remember labels are indexed so the `traceId` here can overwhelm loki.